Tests/Unit/DependencyInjection/CmfResourceExtensionTest.php

```diff
@@ -12,75 +12,32 @@
-use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 
-class CmfResourceExtensionTest extends AbstractExtensionTestCase
+class CmfResourceExtensionTest extends \PHPUnit_Framework_TestCase
 {
-    protected function getContainerExtensions()
-    {
-        return array(new CmfResourceExtension());
-    }
-
-    public function provideExtension()
-    {
-        return array(
-            array(
-                array(
-                    'repositories' => array(
-                        'foobar_odm' => array(
-                            'type' => 'doctrine_phpcr_odm',
-                            'options' => array(
-                                'basepath' => '/cmf/foo',
-                            ),
-                        ),
-                        'foobar_phpcr' => array(
-                            'type' => 'doctrine_phpcr',
-                            'options' => array(
-                                'basepath' => '/cmf/foo',
-                            ),
-                        ),
-                        'foobar_filesystem' => array(
-                            'type' => 'filesystem',
-                            'options' => array(
-                                'base_dir' => '/assets',
-                            ),
-                        ),
-                        'unified' => array(
-                            'type' => 'composite',
-                            'options' => array(
-                                'mounts' => array(
-                                    array(
-                                        'repository' => 'foobar',
-                                        'mountpoint' => '/foobar',
-                                    ),
-                                ),
-                            ),
-                        ),
-                    ),
-                ),
-                array(
-                    'cmf_resource.repository.foobar_odm',
-                    'cmf_resource.repository.foobar_phpcr',
-                    'cmf_resource.repository.unified',
-                    'cmf_resource.repository.foobar_filesystem',
-                ),
-            ),
-            array(
-                array(),
-                array(),
-            ),
-        );
-    }
-
     /**
-     * @dataProvider provideExtension
+     * It should set the repository configuration.
      */
-    public function testExtension($config, $expectedServiceIds)
+    public function testExtension()
     {
-        $this->load($config);
+        $container = new ContainerBuilder();
+        $extension = new CmfResourceExtension();
+        $extension->load([
+            [
+                'repositories' => [
+                    'test' => [
+                        'type' => 'foobar',
+                    ],
+                ],
+            ],
+        ], $container);
```

Comment:
Since this is a `0.x.x` bundle, and this PR updates it to have a minimum PHP version of 5.5 (as with Symfony 3 and the `master` branch of RoutingAuto), what harm could come from using the new syntax? Saying that it seems that Symfony 3 will not adopt the new syntax whilst changes have to be merged from 2.8, so fine with using the old syntax.